Sources/NIO/Bootstrap.swift

```diff
@@ -503,6 +585,19 @@ public final class ClientBootstrap: NIOClientTCPBootstrapProtocol {
         self._channelOptions.append(key: option, value: value)
         return self
     }
+    
+    /// Specifies some `ChannelOption`s to be applied to the `SocketChannel`.
+    /// - See: channelOption
+    /// - Parameter options: List of shorthand options to apply.
+    /// - Returns: The updated client bootstrap (`self` being mutated)
+    @inlinable
+    public func channelOptions(_ options: [NIOTCPShorthandOption]) -> Self {
+        var toReturn = self
+        for option in options {
+            toReturn = option.applyOption(with: toReturn)
```

Comment:
Minor nit: why is this `with:` while the other options are `to:`?

Reply:
I originally envisaged this being done with a function rather than an implemented protocol.  I might have another go at making that work before I give up.

Reply:
All are now the same.